screener.config.js

```diff
@@ -31,26 +37,37 @@ const screenerConfig = {
   states: glob
     .sync('docs/src/examples/**/*.tsx', { ignore: ['**/index.tsx', '**/*.knobs.tsx'] })
     .reduce((states, examplePath) => {
-      const { name: nameWithoutExtension, base: nameWithExtension, dir } = path.parse(examplePath)
-      const rtl = nameWithExtension.endsWith('.rtl.tsx')
-      const url = `http://localhost:8080/maximize/${_.kebabCase(nameWithoutExtension)}/${rtl}`
+      const {
+        name: exampleNameWithoutExtension,
+        base: exampleNameWithExtension,
+        dir: exampleDir,
+      } = path.parse(examplePath)
+
+      const rtl = exampleNameWithExtension.endsWith('.rtl.tsx')
+      const url = `http://${SCREENER_HOST_URL}/maximize/${_.kebabCase(
+        exampleNameWithoutExtension,
+      )}/${rtl}`
```

Comment:
```suggestion
const exampleUrl = _.kebabCase(exampleNameWithoutExtension)
const url = `http://${SCREENER_HOST_URL}/maximize/${exampleUrl}/${rtl}`
```

Should be more readable